### DISCUSSION TITLE
keep only one TA instance open

### DISCUSSION THREAD

**Discussion by ousia** [orbitalquark/textadept#455] [Q&A] [answered]
After moving from version 11.4 to 12 in Windows, I’m experiencing an issue that already lead me to data loss.

Having TA already running and opening (by double clicking) a file which is associated with TA, in version 12 a new instance is opened.

Sorry, I don’t know what has been changed, but it is essential for me that a new TA instance opens when already one has been opened.

Many thanks for your help.

**Comment by orbitalquark** [comment, marked as answer]
Sorry :( There's an open issue about this: https://github.com/orbitalquark/textadept/issues/361. Qt does not provide native, single-instance functionality like GTK does.

**Reply by ousia** [reply]
Commenting there.
